proxy/coco/src/peer.rs

```diff
@@ -259,6 +259,29 @@ impl Api {
         Ok(project_meta)
     }
 
+    /// Returns the list of [`librad_project::Project`]s that you have contributed to.
+    ///
+    /// # Errors
+    ///
+    /// * if opening the storage fails
+    pub fn list_my_projects(&self) -> Result<Vec<librad_project::Project<entity::Draft>>, Error> {
+        let projects = self
+            .list_projects()?
+            .into_iter()
+            .filter(|project| {
+                // TODO(sos): either update list_project_refs() to return an Option
```

Comment:
`Refs` is an iterator, so if there is none the iterator will be empty, i.e. `refs.next().is_none()`